achilles-core/src/test/java/info/archinnov/achilles/LoggerTestUtils.java

```diff
@@ -0,0 +1,31 @@
+package info.archinnov.achilles;
+
+import static info.archinnov.achilles.logger.AchillesLoggers.ACHILLES_DML_STATEMENT;
+import static org.mockito.Mockito.mock;
+import java.lang.reflect.Field;
+import org.apache.log4j.Appender;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.impl.Log4jLoggerAdapter;
+
+public abstract class LoggerTestUtils {
+    public static Appender interceptDMLStatementViaMockedAppender() {
```

Comment:
You are modifiying a shared variable, you should restore its state after the test, maybe with a junit rule